<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.545">
    <properties/>
    <testcase name="delChannel2" time="0.545" classname="meibanStaffChannel.delChannel">
        <failure type="java.lang.AssertionError"
                 message="10500 StateMessage:System error"
        >
            <![CDATA[java.lang.AssertionError: 10500 StateMessage:System error
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.assertNotEquals(Assert.java:930)
  meibanStaffChannel.delChannel.delChannel2(delChannel.java:38)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1191)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1116)
  org.testng.TestNG.run(TestNG.java:1024)
  org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
]]>
        </failure>

    </testcase>


</testsuite>
